src/main/webapp/app/pages/apiAccessGroup/APIAccessPage.tsx

```diff
@@ -82,6 +82,21 @@ const DownloadButtonGroups: React.FunctionComponent<{
           buttonText="Cancer Gene List"
         />
       ) : null}
```

Comment:
Can we also include allActionableVariantsTxt as well. We sometimes get that request too and should be quick to add?